### DISCUSSION TITLE
Organisation purchases github copilot for Members

### DISCUSSION THREAD

**Discussion by grantspeelman** [community/community#20986] [Copilot Conversations] [answered]
Is it possible or are there any plans for an organisation to be able purchase github copilot for it's members?

**Comment by baysideengineer** [comment, marked as answer]
Hey @grantspeelman, it seems there are some plans to do so. GitHub Copilot is coming for companies later this year. You can [sign up](https://github.com/features/copilot/org_signup) for more information. Have a nice day.
